@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -465,7 +465,7 @@ export type StyleSet = StyleDeclaration[];
  * tile. [[Style]] is describing which features are shown on a map and in what way they are being
  * shown.
  */
-export interface BaseStyle {
+export type BaseStyle<Params> = Partial<Omit<Params, "technique">> & {
```

Comment:
Omit seems not needed, we're feeding BaseStyle with interfaces without `technique` attr.

Reply:
removed